pam/internal/adapter/button.go

```diff
@@ -33,6 +33,9 @@ func (b buttonModel) Init() tea.Cmd {
 // Update handles events and actions.
```

Comment:
Typo in the commit message (`s/shoudld/should`):

```
The first time we hit the button we can safely handle the event
promptly, while we shoudld wait when authentication has just started
```

Reply:
Thanks for catching it, will fix at rebase time